### PR TITLE
Strips previous embedded JWT before embedding new one

### PR DESCRIPTION
**Pull request by autodidaddict**
This fixes a bug where embedding signed claims in a file that had previously been signed would actually append a second custom section, thereby making it seem like the file hadn't been signed with the new claims.

The new code strips the previous embedded JWT (if any) prior to signing/embedding.

### CODE REVIEW

File: src/wasm.rs [hunk 1]
```diff
@@ -27,16 +25,17 @@ const SECTION_WC_JWT: &str = "wasmcloud_jwt";
...
+    let target_hash = compute_hash(&strip_custom_section(contents.as_ref())?)?;
     let parser = wasmparser::Parser::new(0);
     for payload in parser.parse_all(contents.as_ref()) {
         match payload? {
             wasmparser::Payload::CustomSection(reader) => {
                 if reader.name() == SECTION_JWT || reader.name() == SECTION_WC_JWT {
                     let jwt = String::from_utf8(reader.data().to_vec())?;
                     let claims: Claims<Actor> = Claims::decode(&jwt)?;
-                    let hash = compute_hash_without_jwt(contents.as_ref())?;
+                    //let hash = compute_hash_without_jwt(contents.as_ref())?;
```
Comment (connorsmith256) [inline src/wasm.rs]: Nit: remove

**Review by connorsmith256 — APPROVED** [review]
Does this preserve hashes from before this change (and does that matter)?

File: src/wasm.rs (hunk 1 above)
Comment (autodidaddict) [reply src/wasm.rs]: Old hashes are still not compatible but it doesn't matter. There's a test to prove that old hashes still pass validation checks.

Comment (connorsmith256) [reply src/wasm.rs]: Sweet, just remember to remove this commented line before merging

**Pull request merged**